src/api/animals/Animals1Dash.js

```diff
@@ -81,6 +81,17 @@ const AnimalDash = () => {
             difficulty="Easy"
           />
         </Grid>
+
+        <Grid item xs={12} sm={6} md={4} lg={3}>
+          <CardRouting
+            cardName="Cat Facts"
+            navigateTo="AnimalCatFacts"

```

Comment:
Updat to AnimalsCatFacts

Reply:
I have made the necessary changes could you please look into it